### ISSUE TITLE
New Component: Conic Gradient

### ISSUE THREAD

**Issue by endigo9740** [Brain-Bones/skeleton#203]
Note this component already exists here:
https://github.com/Brain-Bones/skeleton/blob/dev/src/lib/ConicGradient/ConicGradient.svelte

It's currently incomplete as I was struggling to find a proper means to pass in the color values. The radial gradient doesn't like CSS variables, and Tailwind doesn't currently have a utility class for this (to my knowledge).

It may be possible a third party Tailwind library exists for this, so some research should be done to determine the best means to handle this.

If we cannot find an adaquate solution it may be best to sunset this one. We'll then purge it from the library.

**Comment by endigo9740** [comment]
Accessing default Tailwind colors, the official means:
https://stackoverflow.com/questions/61118060/how-to-access-tailwind-colors-from-javascript

**Comment by endigo9740** [comment]
I've pushed a PR that updates this component to support three methods of adding color stops:

1. Provide `rgb(vars(--color-primary-500))` for theme colors
2. Provide `colors: ['purple',500]` for default Tailwind colors
3. Provide any standard CSS color as a string, including named colors, hex, HSL, etc.

I also updated the documentation and test cases to be current and re-enabled the component in the sidebar navigation.
